### ISSUE TITLE
deb builds create identical filenames

### ISSUE THREAD

**Issue by xdgc** [lastpass/lastpass-cli#416]
Background: I package lastpass-cli for our desktop systems at work. We use three different Ubuntu distros for different roles, but have a single reprepro server to provide them via apt. Reprepro is a bit challenged when managing multiple release codenames, because it can't add a package multiple times with the same package name and version (even if they're for distinct releases). So I need these packages to be built with altered version strings.

I'm pretty debhelper illiterate but this patch accomplishes my goal. It has the side effect of changing ALL builds to add codename to version, which I understand might be undesirable. How would relevant people wish to address that so that the functionality can be merged?

`diff --git a/debian/rules b/debian/rules
index 00444d4..8d8937d 100755
--- a/debian/rules
+++ b/debian/rules
@@ -9,8 +9,14 @@
 # Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1

+VERSION = $(shell git describe --tags | sed -e 's/^v//')
+RELEASE = $(shell lsb_release -sc)
+
 override_dh_auto_build:
 	dh_auto_build -- all doc-man

+override_dh_gencontrol:
+	dh_gencontrol -- "-v$(VERSION)-$(RELEASE)"
+
 %:
 	dh $@

`

**Comment by waterkip** [comment]
I don't think this is an issue lastpass need to fix. Perhaps have a look here: https://serverfault.com/questions/556206/create-debian-repository-with-multiple-packages-verions
